src/app/clusters/keypad-input-server/keypad-input-server.cpp

```diff
@@ -39,11 +39,37 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
-bool emberAfKeypadInputClusterSendKeyCallback(chip::app::Command * commandObj, unsigned char)
+EmberAfKeypadInputStatus keypadInputClusterSendKey(EmberAfKeypadInputCecKeyCode keyCode);
```

Comment:
Callbacks zapt as followup.

Reply:
Issue [link](https://github.com/project-chip/connectedhomeip/issues/8086) for follow up.